conda/environments/raft_dev_cuda11.0.yml

```
@@ -15,7 +15,7 @@ dependencies:
 - rapids-notebook-env=22.02.*
 - rapids-doc-env=22.02.*
 - rmm=22.02.*
-- dask-cuda=22.02.*
+- dask-cuda=22.08.
```

To match other repos' pinnings (and other pinnings in this file), I think this needs a trailing `*`:
```suggestion
- dask-cuda=22.08.*
```